SohamGaikwad/Advanced/NewsApp/app/src/main/java/com/example/newsapp/MyPageAdapter.java

```diff
@@ -0,0 +1,44 @@
+package com.example.newsapp;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentStatePagerAdapter;
+
+public class MyPageAdapter extends FragmentStatePagerAdapter {
+
+
+
+    public MyPageAdapter(FragmentManager fm) {
+
+        super(fm);
```

Comment:
Constructor 'FragmentStatePagerAdapter(androidx.fragment.app.FragmentManager)' is deprecated, you shall use other recommended constructor here.

Reply:
Fixed in ad34fcde338dffceb4d0ffa797d1c1bfc738641e.